kornia/augmentation/_2d/intensity/color_jitter.py

```diff
@@ -66,23 +67,31 @@ def __init__(
         p: float = 1.0,
         keepdim: bool = False,
         return_transform: Optional[bool] = None,
+        pillow_like: bool = False
```

Comment:
i'd like to have a more library name free parameter here like `brightness_mode=["additive", "accumulative"]` which is in the end what's happening. Consider checking this PR I opened a couple of weeks ago: https://github.com/kornia/kornia/pull/1586

Reply:
Thanks for your feedback, in this case it would mean having parameters for brightness, contrast and saturation which offers more flexibility but also adds more verbosity to the ColorJitter class. To ease the user experience doing what you describe plus having a subclass of `ColorJitter` with a name like `ColorJitterPillow` calling super with the pillow arguments might be interesting.

For brightness indeed the list that you propose describes what's happening, for contrast and saturation maybe a list such as `contrast_mode=["opencv", "pillow"]` might be the way to go.

Reply:
for contrast can have the same I believe. We recently had the discussion with @shijianjian that color jitter in the end is a combo of 4 ops kind of arbitrary and very sensitive (you can make infinite color jitters combining stuff). That's also why more advanced things like https://github.com/kornia/kornia/pull/1607 will just replace it in the short term

Reply:
For contrast and saturation do you agree with a list of backends as modes such as `contrast_mode=["opencv", "pillow"]` and `saturation_mode=["opencv", "pillow"]` . It allows the flexibility to add more backends later and solve the issue of forcing RGB or 1-channel image only for the `"pillow"` mode